swift_browser_ui_frontend/src/common/lang.js

```diff
@@ -584,13 +591,14 @@ let default_translations = {
         norename:
           "Säiliötä ei voi nimetä uudelleen, " +
           "mutta sen voi kopioida uudella nimellä.",
-        createdFolder: "",
+        createdFolder:
+          "Created folder will be shared with all project members in",
```

Comment:
Missing translation, use e.g.
> Luotu kansio jaetaan kaikille [My research project]:n jäsenille

Reply:
Alternatively
> Luotu kansio jaetaan kaikille jäsenille projektissa [My research project]

If we don't want to bother with different word ordering in translations.

Reply:
Thanks 👍